src/executor/order_by_executor.cpp

```diff
@@ -57,7 +57,8 @@ bool OrderByExecutor::DExecute() {
   PL_ASSERT(input_tiles_.size() > 0);
 
   // Returned tiles must be newly created physical tiles,
-  // which have the same physical schema as input tiles.
+  // which not always have the same physical schema as input tiles.
```

Comment:
We can rephrase this as: "NOTE: the schema of these tiles might not match the input tiles when some of the order by columns are not be part of the output schema"